### DISCUSSION TITLE
Time Segmented Volume indicator

### DISCUSSION THREAD

**Discussion by whatthejay** [DaveSkender/Stock.Indicators#933] [Ideas]
Thank you for this great library.

It would be great to add Time Segmented Volume (TSV) indicator, which is a leading indicator and helps to decide price trends based on the volume changes in specific periods.

### References
https://www.worden.com/TeleChartHelp/Content/Indicators/Time_Segmented_Volume.htm
https://help.tc2000.com/m/69404/l/747088-time-segmented-volume

### Possible implementation
https://ctrader.com/algos/indicators/show/2750

**Comment by DaveSkender** [comment]
Since this is "a proprietary technical indicator developed by Worden Brothers, Inc.", we wouldn't be able to add it.  They'd have to publish the formula or otherwise grant permission for us to use it openly.
